src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2187,15 +2187,15 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <Field Name="Argument" Type="BoundExpression" Null="disallow"/>
-    <Field Name="ConstantValueOpt" Type="ConstantValue" Null="disallow"/>
```

Comment:
>Null="disallow" [](http://example.com/codeflow?start=56&length=15)

Why did we drop ```Null="disallow"``` part?

Reply:
@alrz Could you comment on this?

Reply:
Nullability is already inferred by the type itself. Annotations are required anyways but usages of `"disallow"` for reference types is not consistent across the file.